index2.html

```diff
@@ -101,32 +106,74 @@ <h2>Glauber Costa in ITNEXT</h2>
       <h2>How to overcome common objections to bike lanes</h2>
       <p>In my 3+ years of local advocacy I've heard of every objection under the sun...</p>
     </div>
+  </div>
 
   <div class="row">
-    <div class="column side jessie-side-content">
-      <p>Barney Abramson in Momentum</p>
-      <h2>Celebrating Hispanic Heritage Month As A First-Gen Afro-Dominicano</h2>
-      <p>Dominicans say "Cojelo con take it easy" instead of saying "Don't worry." and I think that's beautiful.</p>
-    </div>
+    <div class="col-8">
+      <p>Michael Schneider<p>
+      <h2>How to overcome common objections to bike lanes</h2>
+      <p>In my 3+ years of local advocacy I've heard of every objection under the sun...</p>
+      <hr/>
+      <p>Michael Schneider<p>
+      <h2>How to overcome common objections to bike lanes</h2>
+      <p>In my 3+ years of local advocacy I've heard of every objection under the sun...</p>
+      <hr/>
+      <p>Michael Schneider<p>
+      <h2>How to overcome common objections to bike lanes</h2>
+      <p>In my 3+ years of local advocacy I've heard of every objection under the sun...</p>
+      <hr/>
 
-    <div class="row">
-    <div class="column side jessie-side-content">
-      <p>Robert Roy Britt</p>
-      <h2>Top Sleep Myths I Learned while Researching My New Book</h2>
-      <p>Excerpt: Chapter 1 of "Make Sleep Your Superpower"</p>
     </div>
+    <div class="col-4">col-4</div>
 
-    <div class="column middle">
-      <h2>Main Content</h2>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas sit amet pretium urna. Vivamus venenatis velit nec neque ultricies, eget elementum magna tristique. Quisque vehicula, risus eget aliquam placerat, purus leo tincidunt eros, eget luctus quam orci in velit. Praesent scelerisque tortor sed accumsan convallis.</p>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas sit amet pretium urna. Vivamus venenatis velit nec neque ultricies, eget elementum magna tristique. Quisque vehicula, risus eget aliquam placerat, purus leo tincidunt eros, eget luctus quam orci in velit. Praesent scelerisque tortor sed accumsan convallis.</p>
-    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-8">
+      <div class="row">
+        <div class="col-8">
```

Comment:
Notice you have a duplicate, nested `col-8`, as well as a duplicated nested `row`

Reply:
Complete